data-plane/tests/src/main/java/dev/knative/eventing/kafka/broker/tests/core/ConfigUnmarshalling.java

```diff
@@ -0,0 +1,222 @@
+package dev.knative.eventing.kafka.broker.tests.core;
+
+import static dev.knative.eventing.kafka.broker.core.file.FileWatcher.FileFormat;
+
+import com.google.protobuf.util.JsonFormat;
+import dev.knative.eventing.kafka.broker.core.config.BrokersConfig.Broker;
+import dev.knative.eventing.kafka.broker.core.config.BrokersConfig.Brokers;
+import dev.knative.eventing.kafka.broker.core.config.BrokersConfig.Trigger;
+import dev.knative.eventing.kafka.broker.core.file.FileWatcherTest;
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.UUID;
+import joptsimple.internal.Strings;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@Fork(value = 1, warmups = 1)
+public class ConfigUnmarshalling {
+
+  private static final Brokers noBrokers = Brokers.newBuilder().build();
+  private static final Brokers full;
+  private static final int LONGEST_NAME = 253;
+  private static final int LONGEST_BROKER_NAME = LONGEST_NAME;
+  private static final int LONGEST_NAMESPACE_NAME = LONGEST_NAME;
+  private static final String K8S_SERVICE_SUFFIX = "svc.cluster.local"; // usual case
+
+  static {
```

Comment:
This static block, together with the alloc of `noBrokers`, should live in a state. If you alloc in a static, the jvm might perform some fancy optimizations on it

Reply:
What about encapsulating both in a static method and call this static method in the constructor of state?